### ISSUE TITLE
Logging info about exception

### ISSUE THREAD

**Issue by CrazzyMann99** [NLog/NLog#4574]
Hello everybody! I have just started to use NLog library. It writes my user messages to my log file in my project and i have no problem with the library, it works good. Now I would like to try write to the log file my text messages and exception information at the same time. Class Logger has that parameters in its method so i guess method Error(Exception ex, string Message) is supposed to do that. The code below write text "Message text" to the file, but writes nothing about the Exception and i don't know why.  What is wrong?

`Logger mylogger = LogManager.GetLogger("loggername");
string Message = "Message text";
Exception ex = new Exception();
mylogger.Error(ex,Message);`

**Comment by snakefoot** [comment]
Yes the default NLog Layout is not perfect, but it will be improved with NLog 5.0 that will include exception-details by default.

Until NLog 5.0 is fully baked, then you can update the `Layout` for you NLog-target, so it will include the exception-details:

Ex.
```xml
<?xml version="1.0" encoding="utf-8" ?>
<nlog>
    <targets>
        <target name="logfile" xsi:type="File"
                fileName="file.txt"
                layout="${longdate}|${level:uppercase=true}|${logger}|${message}${exception:format=tostring}" />
    </targets>

    <rules>
        <logger name="*" minlevel="Debug" writeTo="logfile" />
    </rules>
</nlog>
```

**Comment by CrazzyMann99** [comment]
Thank you. That works.

**Comment by snakefoot** [comment]
Closing issue as resolved.
